hubot-rocketchat/Dockerfile

```diff
@@ -22,23 +22,26 @@ RUN apk --update add \
     coffeescript \
     generator-hubot \
     yo \
- && adduser -D hubot \
- && cd /home/hubot \
- && sudo -u hubot yo hubot --owner="$BOT_OWNER" --name="$BOT_NAME" --description="$BOT_DESC" --defaults \
+ && mkdir /hubot \
+ && cd /hubot \ 
+ && mkdir -p /root/.config/insight-nodejs \
+ && mkdir -p /root/.npm/_cacache \
+ && chmod g+rwx /root /root/.config /root/.config/insight-nodejs /bin /hubot /root/.npm /root/.npm/_cacache \
+ && yo hubot --owner="$BOT_OWNER" --name="$BOT_NAME" --description="$BOT_DESC" --defaults \
  && sed -i /heroku/d ./external-scripts.json \
  && sed -i /redis-brain/d ./external-scripts.json \
- && sudo -u hubot npm install hubot@$HUBOT_VERSION \
```

Comment:
Also remove `sudo` from the list of the packages above, because it's not needed anymore.